src/explore-education-statistics-common/src/components/form/Formik.tsx

```diff
@@ -52,6 +56,13 @@ class Formik<FormValues = FormikValues> extends BaseFormik<FormValues> {
     });
   };
 
+  public preSubmit = () => {
```

Comment:
```
tsc error: Property 'preSubmit' does not exist on type 'Readonly<FormikConfig<FormValues>> & Readonly<{ children?: ReactNode; }>'.
```

How can I append a type onto this? or is there a better solution

Reply:
Adding a `preSubmit` to Formik is a pretty bad idea as it's already dirty enough that we're having to extend the API anyway. 

The other day I suggested that we should put a `onClick` on the submit button itself, I would suggest we explore that further instead of doing this current approach.